run.js

```diff
@@ -106,6 +106,16 @@ export async function run( {
 					}
 					break;
 				}
+				case "error": {
+					const reportId = message.id;
+					const report = reports[ reportId ];
+					touchBrowser( report.browser );
+					reportError( message.data );
+					pendingErrors[ reportId ] ??= Object.create( null );
+					pendingErrors[ reportId ][ message.data.message ] =
+						`${ message.data.message }\n${ message.data.stack }`;
```

Comment:
I think we could show the error name (`ReferenceError` etc.) a change here as well. Right now, for a bad top-level `foo()` call we'll see:
> 1. foo is not defined

instead of the preferred:
> 1. ReferenceError: foo is not defined

Reply:
This is now sharing the exact message from `reportError`, similar to what `testEnd` does.